org.diffoscope.Diffoscope.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: org.diffoscope.Diffoscope
+runtime: org.freedesktop.Sdk
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: diffoscope
+finish-args:
+  - --filesystem=host
+modules:
+  - name: openjpeg
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_CODEC=OFF
+    cleanup: /lib/openjpeg-2.3
+    sources:
+      - type: git
+        url: https://github.com/uclouvain/openjpeg.git
+        commit: d2205ba2ee78faeea659263383446c4472b1f9df
```

Comment:
I'd prefer you use the release tarball and manually include the patch you needed. Being a security sensitive library that makes everything easier to track.

Reply:
I cargo-culted this from Evince https://github.com/flathub/org.gnome.Evince/blob/1ff7c51d47ae268593353ef726b227acc506556d/org.gnome.Evince.json#L54-L56 but I now see that many other apps do use the 2.3.0 tarball.

However, if you can find a release tarball more recent than October 2017 https://github.com/uclouvain/openjpeg/releases which contains (from a quick skim of the commit history) these suspiciously security-sensitive patches from master https://github.com/uclouvain/openjpeg/commit/24d08ff94ad8b0f51534e46f87cf5a2f2f85d22a https://github.com/uclouvain/openjpeg/commit/da5e897232ef824daf9a492e746ed22cf2a43f18 I'm sure many people would be interested to hear it :)

Reply:
Yes, I used that commit in Evince for just that reason...